pkg/model/insight.proto

```diff
@@ -56,3 +57,44 @@ enum InsightApplicationCountLabelKey {
   KIND = 0;
   ACTIVE_STATUS = 1;
 }
+
+// time is treated as unix time second
+
```

Comment:
```suggestion
```

Reply:
Web uses unix time milli second and server uses unix time second. So I want to write this somewhere.

Reply:
then it should be written directly before time fields, not at the head of the file, I guess.

Reply:
@Hosshii Let's fix this and merge this PR.

Instead of writing the comment at this, we should add the comment for each time field as @khanhtc1202 commented.

For example,

```
// Time is the number of seconds since Unix epoch.
int64 from = 2 [(validate.rules).int64.gte = 0];
```